testing/environments/docker/elasticsearch/Dockerfile-5.0.0-alpha4

```diff
@@ -5,7 +5,7 @@ ENV ELASTICSEARCH_VERSION master
 
 
 COPY setup.sh /
-RUN bash setup.sh http://download.elastic.co/elasticsearch/staging/5.0.0-alpha4-b0da471/org/elasticsearch/distribution/deb/elasticsearch/5.0.0-alpha4/elasticsearch-5.0.0-alpha4.deb
+RUN bash setup.sh http://download.elastic.co/elasticsearch/staging/5.0.0-alpha5-95ae58b/org/elasticsearch/distribution/deb/elasticsearch/5.0.0-alpha5/elasticsearch-5.0.0-alpha5.deb
```

Comment:
I initially added alpha5 files but revert it as not all alpha5 builds are available yet, so I only updated the snapshots. Forgot to revert this one :-(